source/parameters/bend.md

```diff
@@ -94,7 +94,7 @@ The field integral for the entrance pole face `edge_int1` is given by
   B_y(s) = \frac{B_{y0}}{1 + \exp[P(s)]}
   ```
   ```{math}
-  P(s) = C_0 + C_1 , s + C_2 , s^2 + C_3 , s^3 + , \ldots
+  P(s) = C_0 + C_1 \cdot s + C_2 \cdot s^2 + C_3 \cdot s^3 + \ldots
```

Comment:
@DavidSagan I am sorry, forgot to ask here: did you mean to write multiplication here or truly a `,`? Hope I did not mess it up.

Reply:
Yes I did mean multiplication.